WG-TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+# OCI Working Group Template
+
+The following is a template that the TOB and/or proposers of a new OCI Working Group may use to document the intended scope and activities of the OCI Working Group following its establishment. {Bracketed sections below} should be filled in with the Working Group's specific details.
+
+# {WG NAME} OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the {WG NAME} Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”). The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter. Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+{Briefly describe the purpose of the WG.}
+
+## Scope
+
+* {Describe the activities and work that are in scope for the WG.}
+
+## Out of Scope
+
+* {Where helpful for clarity or to avoid confusion, describe any activities or topics that are out of scope for the WG.}
+
+## Intended work product
+
+* {Describe the intended work product and outputs of the WG, particularly indicating whether they consist of specification(s) and/or project(s).}
+
+## Governance
+
+* **Working Group**:
+  * The TOB is establishing the WG as an OCI Working Group, pursuant to section 6(p) of the OCI Charter.
```

Comment:
```suggestion
  * The TOB is establishing the WG as an OCI Working Group, pursuant to [section 6(p)](https://github.com/opencontainers/tob/blob/master/CHARTER.md#6-technical-oversight-board-tob) of the OCI Charter.
```
And note that section p is not in the current document, it's defined in this PR.

Reply:
Thank you! I've incorporated this in a couple of places in the follow-up commit.